notebooks/official/model_evaluation/automl_text_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        endpoints = aip.Endpoint.list(
</pre><p>Don't use list to find resources. Use the ID or object directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
These resources are created inside a pipeline and hence cannot be accessed outside the pipeline directly